gradoop-flink/src/main/java/org/gradoop/flink/model/api/tpgm/TemporalGraphOperators.java

```diff
@@ -41,6 +42,26 @@
   // Unary Operators
   //----------------------------------------------------------------------------
 
+  /**
+   * Compares two snapshots of this graph. Given two temporal predicates, this operation
+   * will check if a graph element (a vertex or an edge) was added, removed or kept in the snapshot
+   * based on the second predicate.
```

Comment:
Add a note, that the union of all elements of both snapshots are returned, annotated with the properties mentioned. Elements that do not exist in both snapshots are discarded.